src/components/Footer.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+
+function Footer(props) {
```

Comment:
As you dont give any props to the footer you dont need the props there:

`function Footer() {`